pyproject.toml

```
@@ -0,0 +1,54 @@
+[tool.poetry]
+name = "mcstatus"
+version = "6.5.0"
```

I did a tar-diff on the 6.5.0 sdist and the sdist produced by poetry. There are enough differences that I plan to bump this to 7.0.0.

```suggestion
version = "7.0.0"
```